modules/035-cni-flannel/images/flanneld/main.go

```diff
@@ -75,7 +75,9 @@ func main() {
 		log.Fatal(err)
 	}
 
-	flannelArgs := os.Args[1:]
+	var flannelArgs []string
+	flannelArgs = append(flannelArgs, "flannel")
```

Comment:
i think if you run ```err = unix.Exec("/opt/bin/flanneld", flannelArgs, os.Environ())``` than you should set as flannelArgs[0] `/opt/bin/flanneld` nor `flannel`

Reply:
```
zuzzas@laptop:~$ type sleep 500
sleep is /usr/bin/sleep
zuzzas@laptop:~$ sleep 500
zuzzas@laptop:~$ ps axu | grep 'sleep 500'
zuzzas   1504021  0.0  0.0   8372   932 pts/37   S+   10:03   0:00 sleep 500
```
It's not `/usr/bin/sleep`.

Reply:
```
// Exec calls execve(2), which replaces the calling executable in the process
// tree. argv0 should be the full path to an executable ("/bin/ls") and the
// executable name should also be the first argument in argv (["ls", "-l"]).
// envv are the environment variables that should be passed to the new
// process (["USER=go", "PWD=/tmp"]).
```
You are right, but  if binary name flanneld not flannel :)